src/client/components/SearchAPI/Fetcher.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable react/prop-types */
+import React, { useState, useEffect } from 'react';
+
+const baseUrl = 'https://api.github.com/search/users?q=';
```

Comment:
I get that you were testing using this, but please do not commit as Github API is irrelevant.

Reply:
Should I remove this API and wait for a real one?

Reply:
You should remove API from the component in general. Feel free to mock it in story so that we can test the component. But once the API is built, someone will connect your component to it in a container file.